src/test/resources/institution-name/example-wikidata.json

```diff
@@ -0,0 +1 @@
+"De Museumfabriek"
```

Comment:
Do you need to read this from a file? Why not `MAPPER.readTree(""" ... """)`

Reply:
Yes could as well. Tried to keep it in line with the ror but you are right might be better to change